packages/core/src/PasteRule.ts

```diff
@@ -167,6 +177,45 @@ export function pasteRulesPlugin(props: { editor: Editor; rules: PasteRule[] }):
   let pasteEvent = typeof ClipboardEvent !== 'undefined' ? new ClipboardEvent('paste') : null
   let dropEvent = typeof DragEvent !== 'undefined' ? new DragEvent('drop') : null
 
+  const processEvent = ({
+    state,
+    from,
+    to,
+    rule,
+    pasteEvt,
+  }: {
+    state: EditorState
+    from: number
+    to: { b: number }
```

Comment:
Any reason why to is not just the number too?

Reply:
> Any reason why to is not just the number too?

Yeah, this is a lil dirty. This is what `findDiffX()` returns [here](https://github.com/ueberdosis/tiptap/pull/5046/files#diff-6eddae5591c39b131ab5597e8ac574df71fafda96023c4573c45fd73ed066b9fR289). `{ a: number; b: number }`